addon/components/nrg-appbar/template.hbs

```diff
@@ -10,7 +10,13 @@
       </a>
     {{/if}}
     <div class="header item title-item">
-      <span class="title-text">{{title}}</span>
+      <span class="title-text">
```

Comment:
Instead of having a loading indicator, can we have it so the it doesn't display anything at app launch? After app launch, the environment shows up correctly instead of the flash.

Reply:
Without the loading indicator, that's exactly what happens. That particular section of the appbar is empty until the title is loaded. I'll remove the loading indicator now